website/route-lockfile.txt

```diff
@@ -126,7 +126,38 @@
 /en/developing/developer-faqs/
 /en/developing/supported-networks/
 /en/developing/unit-testing-framework/
-/en/firehose/
+/en/firehose/README/
```

Comment:
We should add a redirect from `/en/firehose/` to `/en/firehose/README/`. This should ideally be done in 2 places:

- In `nginx.conf`, for staging and production
- In `next.config.js`, for local dev with `pnpm dev`

Reply:
@benface added an issue to unblock merging current PR